pkg/utils/selector.go

```diff
@@ -64,18 +64,18 @@ func SelectAndFilterPods(ctx context.Context, c client.Client, spec SelectSpec)
 		return nil, err
 	}
 
-	filteredPod, err := FilterPodsByMode(pods, mode, value)
+	filteredPod, err := filterPodsByMode(pods, mode, value)
 	if err != nil {
 		return nil, err
 	}
 
 	return filteredPod, nil
 }
 
-// SelectPods returns the list of pods that are available for pod chaos action.
+// selectPods returns the list of pods that are available for pod chaos action.
 // It returns all pods that match the configured label, annotation and namespace selectors.
 // If pods are specifically specified by `selector.Pods`, it just returns the selector.Pods.
-func SelectPods(ctx context.Context, c client.Client, selector v1alpha1.SelectorSpec) ([]v1.Pod, error) {
+func selectPods(ctx context.Context, c client.Client, selector v1alpha1.SelectorSpec) ([]v1.Pod, error) {
```

Comment:
'podchaos/podkill/types' and 'podchaos/containerkill/types'  all reference this function.  So the first letter should be capitalized.

Reply:
As they all share the logic of the existing `SelectAndFilterPods`, I've replaced the occurrence in 'podchaos/podkill/types' and 'podchaos/containerkill/types' to `SelectAndFilterPods`, that's why I changed those two function to local.